.github/workflows/lint_python.yml

```
@@ -8,23 +7,15 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
       - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
-      # Security
-      # - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
-
-      # Code style
-      # - run: black --check . || true
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
       - run: codespell || true  # --ignore-words-list="" --skip=""
       - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
       - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --show-source --statistics
-      # order libraries
-      # - run: isort --check-only --profile black . || true
-      
-      # Unit tests
-      # - run: pip install -r requirements.txt || true
-      # - run: mypy --install-types --non-interactive . || true
-      # - run: pytest . || true
-      # - run: pytest --doctest-modules . || true
-      # - run: shopt -s globstar && pyupgrade --py36-plus **/*.py || true
-
-      # - run: safety check  # checks Python dependencies for known safety vulnerabilities and suggests the proper remediations for vulnerabilities detected.
-      
+      - run: isort --check-only --profile black . || true
+      - run: pip install -r requirements.txt || true
+      - run: mypy --install-types --non-interactive . || true
+      - run: pytest . || true
+      - run: pytest --doctest-modules . || true
+      - run: shopt -s globstar && pyupgrade --py36-plus **/*.py || true
+      - run: safety check
```

```suggestion
      # order libraries
      # - run: isort --check-only --profile black . || true

      # Unit tests
      # - run: pip install -r requirements.txt || true
      # - run: mypy --install-types --non-interactive . || true
      # - run: pytest . || true
      # - run: pytest --doctest-modules . || true
      # - run: shopt -s globstar && pyupgrade --py36-plus **/*.py || true

      # - run: safety check  # checks Python dependencies for known safety vulnerabilities and suggests the proper remediations for vulnerabilities detected.

```
Let's keep it commented for now